cpanspec

```diff
@@ -1361,6 +1361,8 @@ END
     if ($config->{patches}) {
         $counter = 0;
         for my $p (sort keys %{$config->{patches}}) {
+            my $args = $config->{patches}->{$p} || '';
+            print $spec "# $1\n" if $args =~ / ?(PATCH-FIX.*)/;
```

Comment:
Can you give an example of how such a line would look like?

Reply:
hallo.patch: -p1 PATCH-FIX-UPSTREAM whatever....

Due to the file format comments above the line are stripped, so I needed to use this form.